### PR TITLE
ci: bump Sage

### PR DESCRIPTION
**Pull request by odsod**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -12,23 +12,27 @@ jobs:
...
-      - name: Fetch tags
-        run: git fetch --force --tags
+      - name: Run semantic-release
+        run: make semantic-release repo=${{ github.repository }} dry=false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run goreleaser
-        uses: goreleaser/goreleaser-action@v3.1.0
+  goreleaser:
```
Comment (ericwenn) [inline .github/workflows/release.yml:26]: Why not in the same job as release?

File: .github/workflows/review.yml [hunk 2]
```diff
@@ -2,14 +2,27 @@ name: review
...
+
+      - name: Dry-run semantic-release
+        run: make semantic-release repo=${{ github.repository }} dry=true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Dry-run GoReleaser
+        run: make go-releaser snapshot=true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ericwenn) [inline .github/workflows/review.yml:28]: What issues would we catch by running this in review?

Comment (odsod) [reply .github/workflows/review.yml:28]: That it can be built without error for all target architectures

**Review by ericwenn — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (ericwenn) [reply .github/workflows/release.yml:26]: Gotcha, this is fine!

**Pull request merged**